nlyte-worker/src/main/java/com/vmware/flowgate/nlyteworker/scheduler/job/common/HandleAssetUtil.java

```diff
@@ -290,6 +323,13 @@ public Asset supplementMaterial(Asset asset,int materialID,
             exsitingAsset.setSubCategory(asset.getSubCategory());
             exsitingAsset.setLastupdate(System.currentTimeMillis());
             exsitingAsset.setMountingSide(asset.getMountingSide());
+            if (exsitingAsset.getCategory().name().equals(AssetCategory.Cabinet.name())) {
```

Comment:
We can use categrory to compare directly instead of use the name.  Since it's enum.